packages/scene-composer/src/layouts/SceneLayout/SceneLayout.tsx

```diff
@@ -47,52 +47,46 @@ const R3FWrapper = (props: {
   children?: any;
   sceneLoaded?: boolean;
 }) => {
+  const { children, sceneLoaded, enableMatterport, matterportConfig } = props;
   const sceneComposerId = useContext(sceneComposerIdContext);
   const ContextBridge = useContextBridge(LoggingContext, sceneComposerIdContext, ThemeContext);
 
-  if (!props.sceneLoaded) {
-    return null;
-  }
-
-  if (props.enableMatterport && props.matterportConfig?.modelId) {
-    return (
-      <MatterportViewer
-        assetBase={props.matterportConfig?.assetBase}
-        m={props.matterportConfig?.modelId}
-        applicationKey={props.matterportConfig?.applicationKey}
-        connect-auth={props.matterportConfig?.accessToken}
-        connect-provider='iot-twinmaker'
-        onReady={(matterportSdk: MpSdk) => {
-          // propagate this out elsewhere if you wish to useMatterportSdk() interface
-          // to control this viewer from other non-3d ui
-          // <MpSdkContext.Provider value={matterportSdk}>
-          //  <Other2DComponents/>
-          // <MpSdkContext.Provider/>
-          console.log('MatterportViewer SDK ready!', matterportSdk);
-          setMatterportSdk(sceneComposerId, matterportSdk);
-        }}
-        style={{ width: '100%', height: '100%' }}
-        search={0}
-        title={0}
-        mt={0}
-        play={1}
-        mls={1}
-      >
-        <ContextBridge>
-          <Suspense fallback={null}>{props.children}</Suspense>
-        </ContextBridge>
-      </MatterportViewer>
-    );
-  } else {
-    setMatterportSdk(sceneComposerId, undefined);
```

Comment:
this line is removed? we probably won't be able to go back to standard scene without this line

Reply:
Good catch. Need to add this line back.

Reply:
This line was creating a console error within React.  Does it not initialize as `undefined`? 

The Block that this was included in was not rendering anything from what I could tell. Once the value comes in for `loadMatterPort` then it will trigger a re-render & return the condition that instead passes `matterportSdk`. 

Can one of you show me how to test for this scenario? 
"we probably won't be able to go back to standard scene without this line"

I'd like to make sure that I'm fully testing this.

Reply:
actually I forgot to comment that line during later testing.. and I can see issue by
1. create a scene with camera view and matterport setup, observe the cameras dropdown on the top bar
2. load the matterport view and observe the dropdown is removed
3. remove the matterport model id from setting, the scene goes back to standard one, but the cameras dropdown is not coming back

Reply:
@mumanity and I worked on this and have added a fix.